### ISSUE TITLE
git-pw: Internal error, you found a bug in Patchwork!

### ISSUE THREAD

**Issue by avagin** [dlespiau/patchwork#210]
```
[root@patchwork patchwork]# git bisect bad
deb22b51be7fa75b01e43038c2fbdfb4072dfab3 is the first bad commit
commit deb22b51be7fa75b01e43038c2fbdfb4072dfab3
Author: Arkadiusz Hiler <arkadiusz.hiler@intel.com>
Date:   Tue Aug 15 14:59:58 2017 +0300

    rest: Use "human name" for the series
    
    Currently all series without a cover letter are visible both through the
    API and the web interface (which uses the API) as a
    "Series without cover letter".
    
    That makes the series hard to identify and takes a toll on developers'
    and maintainers' time.
    
    We already have means of generating friendlier titles in form
    "series starting with THE FIRST PATCH NAME"
    that was created for seeding out results,
    
    So let's reuse it in the API for consistency.
    
    Also bump query number in one of the related tests, became this patch
    introduces an additional one.
    
    Signed-off-by: Arkadiusz Hiler <arkadiusz.hiler@intel.com>
```

**Comment by avagin** [comment]
```
$ git show --pretty=oneline
ba064623ee723548c2ce0ed46fd8c86df3a05308 travis: Workaround mysql issue

$ ../patchwork/git-pw/git-pw list --updated-since 2017-08-26 -j
fatal: Internal error, you found a bug in Patchwork!
```

**Comment by ivyl** [comment]
I've stumbled upon the same issue just yesterday. Fix in on the way.

Thanks!

**Comment by ivyl** [comment]
Fixed with 21f8e2e47458a1e6bccadb292f3237dcbb867244

Let me know it it also works for you so we can close the issue :-)

**Comment by avagin** [comment]
It works for me too. Thank you
